dashboards/persistentvolumesusage.libsonnet

```diff
@@ -1,173 +1,203 @@
-          'volume',
-          '$datasource',
-          'label_values(kubelet_volume_stats_capacity_bytes{%(clusterLabel)s="$cluster", %(kubeletSelector)s, namespace="$namespace"}, persistentvolumeclaim)' % $._config,
-          label='PersistentVolumeClaim',
-          refresh='time',
-          sort=1,
-        )
-      )
-      .addRow(
-        row.new()
-        .addPanel(sizeGraph)
-        .addPanel(sizeGauge)
-      )
-      .addRow(
-        row.new()
-        .addPanel(inodesGraph)
-        .addPanel(inodeGauge)
-      ),
+        tsInodes:
+          tsPanel.new('Volume inodes Usage')
+          + tsPanel.standardOptions.withUnit('none')
+          + tsPanel.queryOptions.withTargets([
+            prometheus.new('${datasource}', 'sum without(instance, node) (topk(1, (kubelet_volume_stats_inodes_used{%(clusterLabel)s="$cluster", %(kubeletSelector)s, namespace="$namespace", persistentvolumeclaim="$volume"})))' % $._config)
+            + prometheus.withLegendFormat('Used inodes'),
+
+            prometheus.new('${datasource}', |||
+              (
+                sum without(instance, node) (topk(1, (kubelet_volume_stats_inodes{%(clusterLabel)s="$cluster", %(kubeletSelector)s, namespace="$namespace", persistentvolumeclaim="$volume"})))
+                -
+                sum without(instance, node) (topk(1, (kubelet_volume_stats_inodes_used{%(clusterLabel)s="$cluster", %(kubeletSelector)s, namespace="$namespace", persistentvolumeclaim="$volume"})))
+              )
+            ||| % $._config)
+            + prometheus.withLegendFormat('Free inodes'),
+          ]),
+        gaugeInodes:
+          gaugePanel(
+            'Volume inodes Usage',
+            'percent',
+            |||
+              max without(instance,node) (
+              topk(1, kubelet_volume_stats_inodes_used{%(clusterLabel)s="$cluster", %(kubeletSelector)s, namespace="$namespace", persistentvolumeclaim="$volume"})
+              /
+              topk(1, kubelet_volume_stats_inodes{%(clusterLabel)s="$cluster", %(kubeletSelector)s, namespace="$namespace", persistentvolumeclaim="$volume"})
+              * 100)
+            ||| % $._config
+          )
+          + gauge.standardOptions.withMin(0)
+          + gauge.standardOptions.withMax(100)
+          + gauge.standardOptions.color.withMode('thresholds')
+          + gauge.standardOptions.thresholds.withMode('absolute')
+          + gauge.standardOptions.thresholds.withSteps(
+            [
+              gauge.thresholdStep.withColor('green')
+              + gauge.thresholdStep.withValue(0),
+
+              gauge.thresholdStep.withColor('orange')
+              + gauge.thresholdStep.withValue(80),
+
+              gauge.thresholdStep.withColor('red')
+              + gauge.thresholdStep.withValue(90),
+            ]
+          ),
+      };
+
+      g.dashboard.new('%(dashboardNamePrefix)sPersistent Volumes' % $._config.grafanaK8s)
+      + g.dashboard.withUid($._config.grafanaDashboardIDs['kubelet.json'])
```

Comment:
```suggestion
      + g.dashboard.withUid($._config.grafanaDashboardIDs['persistentvolumesusage.json'])
```

Reply:
fixed in 1f2610c

Reply:
Ohh, great catch, you caught me copy/pasting 🤦‍♂️